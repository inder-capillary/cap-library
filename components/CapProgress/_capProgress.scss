@import '../styles/_variables.scss';
.cap-progress-showcase{
    width:150px;
}

.ant-progress-circle-path {
    stroke: $CAP_PURPLE !important;
}
.ant-progress-bg{
    background-color:$CAP_PURPLE ;
}