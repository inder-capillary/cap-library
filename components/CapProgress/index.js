/**
*
* CapProgress
*
*/

import React from 'react';
import './_capProgress.scss';
// import styled from 'styled-components';
import { Progress } from "antd";
import PropTypes from 'prop-types';
const classNames = require('classnames');

class CapProgress extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    const {className, ...rest} = this.props;
    return (
      <Progress {...rest} className={classNames("cap-progress-v2", className)} />
    );
  }
}

CapProgress.propTypes = {
  className: PropTypes.string,

};

export default CapProgress;
