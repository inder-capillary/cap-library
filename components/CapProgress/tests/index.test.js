import renderer from 'react-test-renderer';
import React from 'react'; // eslint-disable-line no-unused-vars

import CapProgress from '../index';

describe('<CapProgress />', () => {
  it('CapProgress renders correctly', () => {
    const tree = renderer.create(<CapProgress />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
