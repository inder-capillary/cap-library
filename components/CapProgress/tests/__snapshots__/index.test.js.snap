// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapProgress /> CapProgress renders correctly 1`] = `
<div
  className="ant-progress ant-progress-line ant-progress-status-normal ant-progress-show-info ant-progress-default cap-progress-v2"
>
  <div>
    <div
      className="ant-progress-outer"
    >
      <div
        className="ant-progress-inner"
      >
        <div
          className="ant-progress-bg"
          style={
            Object {
              "background": undefined,
              "borderRadius": "100px",
              "height": 8,
              "width": "0%",
            }
          }
        />
      </div>
    </div>
    <span
      className="ant-progress-text"
      title="0%"
    >
      0%
    </span>
  </div>
</div>
`;
